docs/AQL/master03-syntax.adoc

```diff
@@ -80,38 +81,55 @@ WHERE
 
 This example defines only one AQL variable: `o`, which represents `OBSERVATIONs` that conform to (i.e. created using) the `openEHR-EHR-OBSERVATION.blood_pressure.v1` archetype.
 
+The combination of a variable reference and a subordinate <<_openehr_path_syntax,path>> forms one kind of _identified path_ (see <<AQL Identified Paths>>), used to specify a fine-grained data element within a larger object matched by the `FROM` clause.
+
+[#_variable_syntax]
```

Comment:
I normally do these anchors in the standard form `[[variable syntax]]`, or you can do `[[_variable_syntax]]`, which will generate an HTML xref of the form _variable_syntax. See https://asciidoctor.org/docs/user-manual/#anchordef . I would suggest to not use the '#' even though it is an allowed shorthand, because it's confused with the '#' HTML fragment marker that is generated. You also don't need to put any anchor before a heading. A heading with the text 'My Dog' will generate an auto-anchor '_my_dog', which you can refer to using `<<_my_dog>>` .

Reply:
About headings, I used these manually refs because my intention was to name several subsections as 'Syntax', each part of a parent (e.g. 'Parameters', 'variables', etc). Therefore I needed to manually assign refs, as the autogenerated will be numbered (i.e. _syntax_1, _syntax_2, etc)

Reply:
Ah yes good point - AFAIK, you have to write your own - so for those, I would suggest `[[variable syntax]]` etc.

Reply:
ok, thanks